modules/proxy.js

```diff
@@ -0,0 +1,62 @@
+module.exports = async (ctx) => {
```

Comment:
```suggestion
module.exports = async ({msg,bot,members,cfg}) => {
```
use ES6 structuring (or assign shortcuts early) to avoid having to type `ctx` 47 times throughout this file